app/src/main/java/woowacourse/movie/domain/movieTimePolicy/WeekdayMovieTime.kt

```diff
@@ -0,0 +1,17 @@
+package woowacourse.movie.domain.movieTimePolicy
+
+import java.time.LocalDate
+import java.time.LocalTime
+
+object WeekdayMovieTime : MovieTimePolicy {
+    private val WEEKDAY = 1..5
+    private val SCREEN_TIME = 10 until 24
+    private const val SCREEN_TIME_INTERVAL = 2
+
+    override fun generateTime(date: LocalDate): List<LocalTime>? {
```

Comment:
```suggestion
    override fun generateTime(date: LocalDate): List<LocalTime> {
```
List, Map, Set과 같은 컬렉션은 null 대신 빈 컬렉션을 활용하는 것을 권장합니다. (이하 관련 내용 동일)

Reply:
전부 빈 컬렉션을 반환하도록 변경하였습니다.